projects/addon-doc/package.json

```diff
@@ -33,6 +33,7 @@
         "@taiga-ui/polymorpheus": "^4.6.4",
         "markdown-it": ">=14.1.0",
         "marked": ">=13.0.2",
+        "marked-gfm-heading-id": ">=4.0.0",
```

Comment:
It is used inside `ng-markdown` (we used this library in the past before `marked`)